kernel/include/ext2.h

```diff
@@ -143,7 +143,10 @@ struct ext2_inodetable {
 	uint32_t blocks;
 	uint32_t flags;
 	uint32_t osd1;
-	uint32_t block[15];
+	union {
```

Comment:
Yeah, of the two, that's probably the better approach, but there were fewer references to inode->symlink...